instrumentation/internal/internal-reflection/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/internal/reflection/ReflectionInstrumentationModule.java

```diff
@@ -26,12 +30,15 @@ public boolean defaultEnabled(ConfigProperties config) {
   }
 
   @Override
-  public boolean isIndyModule() {
-    return false;
+  public List<TypeInstrumentation> typeInstrumentations() {
+    return asList(new ClassInstrumentation(), new ReflectionInstrumentation());
   }
 
   @Override
-  public List<TypeInstrumentation> typeInstrumentations() {
-    return asList(new ClassInstrumentation(), new ReflectionInstrumentation());
+  public void injectClasses(ClassInjector injector) {
+    injector
+        .proxyBuilder(
+            "io.opentelemetry.javaagent.instrumentation.internal.reflection.ReflectionHelper")
```

Comment:
can you add a comment here, similar to https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/814c3ea9fadd799bc12c4d268a3ea23ca841218d/instrumentation/graphql-java/graphql-java-20.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/graphql/v20_0/GraphqlInstrumentationModule.java#L42-L44?

Reply:
Fixed in https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/12136/commits/9bcb526bb1e4ba767b853b750fd788ca5e979afa